test-e2e/tests/watch_history/logged_in_test.ts

```diff
@@ -10,41 +9,20 @@ runTestSuite(testConfigs.jwpAuth, testConfigs.jwpAuthNoWatchlist, 'JW Player');
 runTestSuite(testConfigs.cleengAuthvod, testConfigs.cleengAuthvodNoWatchlist, 'Cleeng');
 
 function runTestSuite(config: typeof testConfigs.svod, configNoWatchlist: typeof testConfigs.jwpAuthNoWatchlist, providerName: string) {
-  let loginContext: LoginContext;
-
   Feature(`watch_history - logged in - ${providerName}`).retry(Number(process.env.TEST_RETRY_COUNT) || 0);
 
   Before(({ I }) => {
     I.useConfig(config);
   });
 
-  Scenario(`I can get my watch history when logged in - ${providerName}`, async ({ I }) => {
```

Comment:
Why did you just delete this test?

Reply:
Based on the steps and the changes we made to the test below it it looked like the bottom test included everything tested in the top test

Reply:
Well sure, but as noted below, that test is no longer testing what it says it is. I think we need to test both behaviors, so please restore this one and revert the changes in the one below.